libs/sdk-ui/src/locales.ts

```diff
@@ -16,4 +16,45 @@ export const messages: Record<string, MessageDescriptor> = defineMessages({
     t: { id: "visualization.numericValues.t" },
     p: { id: "visualization.numericValues.p" },
     e: { id: "visualization.numericValues.e" },
+    learnMore: { id: "configurationPanel.drillConfig.learn.more.link" },
```

Comment:
Could you add drillConfig prefix for this message?

Reply:
done